src/methodologies/johnny_decimal/system.py

```
@@ -79,7 +79,7 @@ def initialize_from_directory(self, directory: Path) -> None:
                         self.generator.register_existing_number(number, item)
                         detected_numbers += 1
                         logger.debug(f"Detected number {number.formatted_number} in {item.name}")
-                    except NumberConflictError as e:
+                    except NumberConflictError as e:  # pragma: no cover
                         logger.warning(f"Conflict detected: {e}")
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)